app/assets/stylesheets/application.css.scss

```diff
@@ -220,25 +220,19 @@ td {
 	  width: auto;
 	  padding: 5px;
 		border: 0px;
-		.btn{
+		.btn {
       width: 100%;
     }
-    .btn-default{
+
+    .btn-default {
 	    color: white;
 	    background: black;
 	  }
-	  .champ-img{
+    .champ-img{
```

Comment:
Try to keep your spacing consistent everywhere.  You added a space between the btn class and bracket above, but didn't do it here and other areas.  There should be a space for all css selectors